Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -78,6 +87,10 @@ public struct WebView: UIViewRepresentable {
             withError error: Error
         ) {
             webView.isHidden = false
+            DispatchQueue.main.async {
+                self.parent.isLoading = false
+            }
+            parent.webViewNavDelegate?.showWebViewError()
```

Comment:
what about
```
            DispatchQueue.main.async {
                self.parent.isLoading = false
                self.parent.webViewNavDelegate?.showWebViewError()
            }
```
and remove DispatchQueue.main from the showWebViewError function.
wdyt?

Reply:
Agreed, it's more appropriate I believe. Thanks